machine/quanta/quanta_common_p2020/INSTALL

```diff
@@ -0,0 +1,165 @@
+================================================================================
+  QuantaMesh Common P2020 CPU board
+================================================================================
+
+  Supported Platforms
+=======================
+ - T5032-LY6  (cpu module)
+   32x40GbE QSFP+ ports
+
+ - T3048-LY8  (cpu module)
+   48x1/10GbE SFP+ ports,
+   front 4x40GbE QSFP+ ports, and rear 2x40GbE QSFP+ ports
+
+ - T3048-LY5A (cpu module)
+   48x1/10GbE SFP+ ports and 12x40GbE QSFP+ ports  
+
+ - T3048-LY9  (cpu module)
+   48x100/1000/10GBASE-T,
+   front 4x40GbE QSFP+ ports, and rear 2x40GbE QSFP+ ports
+
+ - T3048-LY9A (cpu module)
+   48x100/1000/10GBASE-T and 6x40GbE QSFP+ ports
+   
+ - T5032-LY6F (cpu module)
+   32x40GbE QSFP+ ports
+
+ - T3024-P05  (cpu on board)
+   24x1/10GbE SFP+ ports and 2x40GbE QSFP+ ports
+
+ - T3024-P05A (cpu on board)
+   24x1/10GbE SFP+ ports
+
+ - For more detail product information, please refer to
+   http://www.quantaqct.com/
+
+  Specification
+===============
+ - Modularized CPU board design
+   1xRJ-45 out-of-band management port (10/100/1000M)
+   1xRJ-45 console port
+   1xmicro SD card
+   1xUSB 2.0 port
+
+  Special Feature
+=================
+ - ONIE recovery mechanism
+   Please refer to ``Recovering ONIE back to factory default``
+
+================================================
+Installing ONIE on Quanta Common P2020 CPU board
+================================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``build-config`` and then
+type ``"make MACHINEROOT=../machine/quanta/ MACHINE=quanta_common_p2020 all"``.
+For example:
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/quanta/ MACHINE=quanta_common_p2020 all
+
+When complete, the ONIE binaries are located in
+``build/images``::
+
+  -rw-r--r-- 4849664 Aug 12 15:39 onie-quanta_common_p2020-r0.bin
+
+Or, change directories to ``machine/quanta/quanta_common_p2020`` to compile ONIE,
+use ``build.sh`` to compile ONIE. For example:
+
+  $ cd machine/quanta/quanta_common_p2020
+  $ ./build.sh all
+
+To compile demo installer.
+
+  $ cd machine/quanta/quanta_common_p2020
+  $ ./build.sh demo
+
+To compile html and pdf documents.
+
+  $ cd machine/quanta/quanta_common_p2020
+  $ ./build.sh docs
```

Comment:
I doubt if this is true any more.  The ONIE project docs are now kept in the project wiki git repo.